Mobile Buy SDK/Mobile Buy SDK Tests/BUYClient+CustomerTests.m

```diff
@@ -424,4 +458,12 @@ - (BUYAccountCredentials *)credentialsForFailure
 	return [BUYAccountCredentials credentialsWithItems:@[email, password]];
 }
 
+#pragma mark - Conveniences -
+
+- (NSString *)randomStringWithLength:(NSInteger)length
+{
+	NSString *UUID = [[NSUUID UUID] UUIDString];
```

Comment:
We're using the first 8 characters for emails (same as android), which should be reliable enough.